src/code/main.c

```diff
@@ -56,6 +59,8 @@ void Main(void* arg) {
     // "System heap initalization"
     osSyncPrintf("システムヒープ初期化 %08x-%08x %08x\n", systemHeapStart, fb, gSystemHeapSize);
     SystemHeap_Init((void*)systemHeapStart, gSystemHeapSize); // initializes the system heap
+
+#ifdef ENABLE_CAMERA_DEBUGGER
```

Comment:
You need to be a little careful here. Although the debug heap is only used for the camera debugger, it can technically be used for other things, so maybe it's best that we leave this code alone for the time being.

Reply:
👍 I wasn't sure if removing the debug heap was a good idea anyway